news/feature-4483.md

```diff
@@ -0,0 +1,5 @@
+`afmongodb`: Bulk MongoDB insert is added via the following options
+
+- `bulk`  (**yes**/no)  turns on [bulk insert ](http://mongoc.org/libmongoc/current/bulk.html)usage, `no` forces the old behavior (each log is inserted one by one into the MongoDB)
```

Comment:
Does bulk sending have any known side effects?

(I'm asking this because we're changing the default value here.)

Reply:
good question, probably we could stay with the old behavior by default.
i guess i switch back to the old one, just to be sure not breaking anything